htdocs/core/lib/pdf.lib.php

```diff
@@ -1287,11 +1288,14 @@ function pdf_pagefoot(&$pdf, $outputlangs, $paramfreetext, $fromcompany, $marge_
 		}
 	}
 	// Show page nb only on iso languages (so default Helvetica font)
-	if (strtolower(pdf_getPDFFont($outputlangs)) == 'helvetica') {
+	// if (strtolower(pdf_getPDFFont($outputlangs)) == 'helvetica') {
```

Comment:
The comment seems to say there is pb with exotic languages.
Did you test for example, greek, chinese, japanse, arab ?

Reply:
https://github.com/Dolibarr/dolibarr/blob/42696954f3c2003eba94dca6e814759397f85077/htdocs/includes/tecnickcom/tcpdf/tcpdf.php#L13965-L13971
drawing this with $pdf->multicell doesn't give something good

actual code (with remove of check if font is helvetica) with dejavusans

![image](https://user-images.githubusercontent.com/3624836/221348151-235b6ab8-c7f3-4c40-b959-5843d14cf26f.png)

with proposed changes

![image](https://user-images.githubusercontent.com/3624836/221348193-36b898fe-33b6-453d-b572-896eaf4d15e4.png)

you need to use `$pdf->PageNo()` and `$pdf->getNumPages()` if you want to draw with `$pdf->MultiCell`
![image](https://user-images.githubusercontent.com/3624836/221348371-40da986a-6d90-4581-b14b-44c7427189fa.png)